app/src/main/java/com/sdp/movemeet/backend/serialization/UserSerializer.java

```diff
@@ -15,18 +16,22 @@
     public static final String PHONE_KEY = "phone";
     // The key used to access the description attribute of a serialized User
     public static final String DESCRIPTION_KEY = "description";
+    // The key used to access the list of registered activities attribute of a serialized User
+    public static final String REGISTERED_ACTIVITIES_KEY = "activityId";
```

Comment:
Wouldn't it be more logical to name this key "activitiesId" (and not "activityId") since the user is certainly going to be registered to many activities?

Reply:
Yes !